web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -929,3 +929,17 @@ function joinup_core_form_views_exposed_form_alter(&$form, FormStateInterface $f
 function joinup_core_preprocess_page(&$variables) {
   $variables['#attached']['library'][] = 'joinup_core/sign_in_destination';
 }
+
+/**
+ * Implements hook_editor_js_settings_alter().
+ *
+ * Strip off the h1, h5, h6 paragraph format options from the 'Format' dropdown.
+ * This does not remove the ability of the user to enter or render these tags.
+ *
+ * @see https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-3682
+ */
+function joinup_core_editor_js_settings_alter(array &$settings) {
+  if (isset($settings['editor']['formats']['content_editor']) && $settings['editor']['formats']['content_editor']['format'] === 'content_editor') {
+    $settings['editor']['formats']['content_editor']['editorSettings']['format_tags'] = preg_replace('/h[1|5|6].*?;/', '', $settings['editor']['formats']['content_editor']['editorSettings']['format_tags']);
```

Comment:
We discussed slightly this already, let's improve this regex, maybe we can explode items to be sure that we don't catch tags that end in `h[1|5|6]`.